src/bootstrap/choices.tpl.html

```diff
@@ -1,6 +1,7 @@
 <ul class="ui-select-choices ui-select-choices-content dropdown-menu"
+    style="display: block"
     role="listbox"
-    ng-show="$select.items.length > 0">
+    ng-show="$select.open">
```

Comment:
I'll back that out, it isn't needed, and results in empty dropdown appearing with the Bootstrap theme.